src/api/schemas/extendedQueryNode.graphql

```diff
@@ -38,43 +87,22 @@ type Channel {
 
 type Category {
   id: ID!
-
   name: String!
-
   videos: [Video!]
 }
 
-type JoystreamMediaLocation {
-  dataObjectId: String!
-}
-
-type HttpMediaLocation {
-  url: String!
-}
-
-# In the future we can add IPFS, Dat, etc.
-union MediaLocation = JoystreamMediaLocation | HttpMediaLocation
-
-type KnownLicense {
-  code: String!
-  name: String
-  description: String
-  url: String
-}
-
-type UserDefinedLicense {
-  content: String!
-}
-
-union License = UserDefinedLicense | KnownLicense
-
-type LicenseEntity {
+type License {
   id: ID!
-  type: License!
+  # Should be Int?
```

Comment:
Yes, as in the schema

Reply:
But I think we will need something like license `title` or `name` and its not available in gist. We have only: 
```graphql
type License implements BaseGraphQLObject {
  id: ID!
  createdAt: DateTime!
  createdById: String!
  updatedAt: DateTime
  updatedById: String
  deletedAt: DateTime
  deletedById: String
  version: Int!
  # License code defined by Joystream
  code: Int
  # Attribution (if required by the license)
  attribution: String
  # Custom license content
  customText: String
  videolicense: [Video!]
}
```
In our schema we used `code` to specify license name: 
![image](https://user-images.githubusercontent.com/51168865/112336858-b945d080-8cbd-11eb-99c9-0bb7a2ebf468.png)

Reply:
Hmm, right, will ask about that

Reply:
https://github.com/Joystream/joystream/blob/sumer/content-metadata-protobuf/src/KnownLicenses.json this is list of know licenses, I guess we should just include it and use for lookup